### PR TITLE
Fix tests affected by string_scan_range escaping backslashes

### PR DESCRIPTION
**Pull request by kazarmy**
Fixed tests affected by radare/radare2#8580.

### CODE REVIEW

File: t/cmd_i [hunk 1]
```diff
@@ -907,7 +907,7 @@ FILE=../bins/elf/strenc
 ARGS=
 CMDS="iz~green
 iz~wall"
-EXPECT='vaddr=0x0040224a paddr=0x0000224a ordinal=005 sz=116 len=56 section=.rodata type=utf16le string=utf16le> \u00a2\u20ac\U00010348 in green:\e[32m ¢€𐍈 \e[0m\n blocks=Basic Latin,Latin-1 Supplement,Currency Symbols,Gothic
+EXPECT='vaddr=0x0040224a paddr=0x0000224a ordinal=005 sz=116 len=56 section=.rodata type=utf16le string=utf16le> \\u00a2\\u20ac\\U00010348 in green:\e[32m ¢€𐍈 \e[0m\n blocks=Basic Latin,Latin-1 Supplement,Currency Symbols,Gothic
```
Comment (radare) [inline t/cmd_i]: looks wrong to me

Comment (kazarmy) [reply t/cmd_i]: ok, humans can probably disambiguate correctly between uses of `\`. Computers are not that smart so I think `izj` should properly escape backslashes but that will be a later PR. I'll close this PR soon.

Comment (kazarmy) [reply t/cmd_i]: Hmmm `izj` should base64 the original string and not the escaped one.

**Comment by kazarmy** [comment]
Closing since radare/radare2#8580 was closed.

File: t/cmd_i (hunk 1 above)
Comment (kazarmy) [reply t/cmd_i]: > looks wrong to me

Fixed.

**Pull request merged**